internal/provisioner/plain/controllers/bundle_controller.go

```diff
@@ -262,11 +300,13 @@ func (r *BundleReconciler) handleCompletedPod(ctx context.Context, u *updater.Up
 		return updateStatusUnpackFailing(u, fmt.Errorf("get bundle contents: %w", err))
 	}
 
+	// TODO: this doesn't apply to all bundle source types
 	bundleImageDigest, err := r.getBundleImageDigest(pod)
 	if err != nil {
 		return updateStatusUnpackFailing(u, fmt.Errorf("get bundle image digest: %w", err))
 	}
-
+	// TODO: what happens when len(objects) == 0? how to avoid silently storing zero manifests
+	// despite them being populated correctly in the Pod logs?
```

Comment:
Rebase against the latest main branch?

Reply:
Hmm I thought I did rebase...checking again

Reply:
Looks like I did but the comment came through anyway...will remove